### PR TITLE
Feature/coverage entities therestofthem

### PR DESCRIPTION
**Pull request by A-Daneel**
alright, I agree. not all of them. But I can't be bothered with the recursion now and deadlines are thing. (and study should also be a thing). I hope the new phpstan won't bend me over though.

### CODE REVIEW

File: tests/Unit/Entity/Group/GroupTest.php [hunk 1]
```diff
@@ -47,7 +48,7 @@ public function testGetId(): void
             ->getProperty('id');
         $property->setAccessible(true);
         $property->setValue($this->group, $expected);
-        $this->assertSame($expected, $this->group->getId());
+        $this::assertSame($expected, $this->group->getId());
```
Comment (jasperweyne) [inline tests/Unit/Entity/Group/GroupTest.php:51]: Should be `self::assert...`, as it's a static

File: src/Entity/Mail/Mail.php [hunk 2]
```diff
@@ -12,39 +13,46 @@
...
 
     /**
+     * @var LocalAccount | null
      * @ORM\ManyToOne(targetEntity="App\Entity\Security\LocalAccount")
      * @ORM\JoinColumn(name="person_id", referencedColumnName="id")
      */
     private $person;
 
     /**
+     * @var Collection<int,Recipient>
```
Comment (jasperweyne) [inline src/Entity/Mail/Mail.php:31]: I'm not sure of this, but I believe this value could be hydrated with type `Collection<int,Recipient>|Recipient[]` (these sort of types occur in more places, also 272)

**Review by jasperweyne — CHANGES_REQUESTED** [review]
Couple of small notes, should be quickly fixable

File: src/Entity/Mail/Mail.php (hunk 2 above)
Comment (A-Daneel) [reply src/Entity/Mail/Mail.php]: could not find any others in this PR, I've made a not for the remaining one's

**Review by jasperweyne — APPROVED** [review]

**Pull request merged**